apps/posters/models.py

```diff
@@ -41,6 +41,7 @@ class OrderMixin(models.Model):
         null=True
     )
     description = models.TextField(_(u"beskrivelse"), max_length=1000, blank=True, null=True)
+    comments = models.TextField(_(u"kommentar"), max_length=1000, blank=True, null=True)
```

Comment:
Why have a max length at all? I suggest removing it from description too. It can never be too much information.